Answers.md

```diff
@@ -1,6 +1,23 @@
 1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects? Which method do we use to create a new object while extending the properties of another object?
+
+map, filter and concat although map and filter seem the most used?
```

Comment:
Yup, those 3 do not produce side-effects and, in addition, the method we use to create a new object while extending the properties of another object is either *Object.assign() or spread syntax.*